pkgs/applications/misc/smb3-foundry/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, makeWrapper
+}:
+let
+  pythonEnv = (python3.withPackages (ps: with ps;[
+    pyside6
+    py65
+    qdarkstyle
+  ]));
+in
+stdenv.mkDerivation rec {
```

I recommend the rec-less version:


```suggestion
stdenv.mkDerivation (finalAttrs: {
```